src/encoded/schemas/sc_atac_analysis_quality_metric.json

```diff
@@ -136,11 +139,12 @@
             }
         },
         "archr_doublet_summary_text": {
-            "title": "ArchR doublet data",
+            "title": "Pre-filtering ArchR doublet data",
```

Comment:
The property title can be kept as the original title.

Reply:
done